x-pack/libbeat/management/generate_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package management
```

Comment:
Should there be unit tests of the individual transformations here? How well are we covering the code in generate.go with these tests?

Reply:
Good idea, might want to add more....